app/controllers/preference_sets_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PreferenceSetsController < ApplicationController
+  def table_state
+    skip_authorization
+    render json: PreferenceSetTableStateService.new(user_id: current_user.id).table_state(
+      table_name: "volunteers_table"
```

Comment:
you'll need to `params[:table_name]` here because that will be passed up (what we discussed last time we talked).

Reply:
Oh yes! my bad! I lost the notes I took and I can't remember the exact route naming your game me but I think this is close. But let me know If you need me to change it! than